gen/generator.go

```diff
@@ -94,10 +94,11 @@ var templateFuncs = template.FuncMap{
 	"containsNoneAttributeValue":   ContainsNoneAttributeValue,
 	"definedInMap":                 DefinedInMap,
 	"add":                          func(val1, val2 int) int { return val1 + val2 },
-	"substract":                    func(val1, val2 int) int { return val1 - val2 },
+	"subtract":                     func(val1, val2 int) int { return val1 - val2 },
```

Comment:
nice catch, totally missed this last review

Reply:
Yeah I caught it in this review but not sure what happened to the PR because it got closed. 
https://github.com/CiscoDevNet/terraform-provider-aci/pull/1240#discussion_r1653734177

Reply:
@samiib Yeah I closed the previous PR because I really screwed up my last rebase hehe. Since I added small changes to the generator, it was easier to create a new PR for time efficiency matters.